cf-deployment.yml

```diff
@@ -35,6 +37,12 @@ addons:
         client_key: "((expvar_forwarder.private_key))"
       default_source_id: metron
       counters:
+      - addr: http://127.0.0.1:14823/debug/vars
```

Comment:
Is this the right port, or should be 14824 like the other ones?

Reply:
cc @chentom88 @MasslessParticle

Reply:
@cdutra Good catch, looks like we had it wrong in our original ops file and it was copied over 

https://github.com/cloudfoundry/cf-deployment/blob/develop/operations/experimental/deploy-forwarder-agent.yml#L64

We'll fix it. Thanks.

Reply:
@cdutra Actually that port is correct. We want this metric to be coming from the new forwarder agent which is exposed at port `14823`.

Reply:
This is the right port; loggregator_agent and forwarder_agent use different ports for metrics because they both spin up an http server to host their metrics and are colocated